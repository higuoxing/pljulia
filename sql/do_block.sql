DO $$ elog("INFO", "Prints an info message") $$ language pljulia;