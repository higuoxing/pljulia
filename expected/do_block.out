DO $$ elog("INFO", "Prints an info message") $$ language pljulia;
INFO:  Prints an info message
